src/components/TokenBottomSheet.tsx

```diff
@@ -293,6 +293,9 @@ function TokenBottomSheet({
           }
           return null
         }}
+        onScrollBeginDrag={() => {
+          Keyboard.dismiss()
```

Comment:
this is unrelated directly to this feature but i noticed it. the keyboard covers the token bottom sheet content instead of reducing the scrollable area

Reply:
Thanks for fixing, I wonder if it's preferable to keep the keyboard up and use the KeyboardSpacer instead so all items are fully visible while the keyboard is up.